lib/logger/lib/logger.ex

```diff
@@ -304,7 +304,8 @@ defmodule Logger do
   end
 
   @doc """
-  Adds the given keyword list to the current process metadata.
+  Adds the given keyword list to the current process metadata. This also allows you to 
```

Comment:
First paragraph need to be concise since it serves as summary.
Maybe we can say something like: "Alters the current process metadata according the given keyword list."
What do you think?